### PR TITLE
Added Controls enum to remove all vc configuration from App delegate

### PR DESCRIPTION
**Pull request by RomanTysiachnik**
## Changes
- It is now easier to select Controls ViewController for ad or content from AppDelegate

@VerizonAdPlatforms/mobile-sdk-developers: Please review.

### CODE REVIEW

File: Demo/Demo/AdControls.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
+//  Copyright 2019, Oath Inc.
+//  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
+
+import PlayerControls
+
+func adControls() -> AdVideoControls {
```
Comment (AndriiMoskvin) [inline Demo/Demo/AdControls.swift]: replace this with global variable?

File: Demo/Demo/ContentControls.swift [hunk 2]
```diff
@@ -0,0 +1,139 @@
+//  Copyright 2019, Oath Inc.
+//  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
+
+import PlayerControls
+
+typealias Props = ContentControlsViewController.Props
+
+func contentControls() -> ContentControlsViewController {
```
Comment (AndriiMoskvin) [inline Demo/Demo/ContentControls.swift]: Same here.

File: Demo/Demo/ContentControls.swift [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+            contentFullScreen: .init(action: backgroundColor),
+            brandedContent: Props.BrandedContent {
+                $0.advertisementText = "Short text"
+                $0.action = .nop
+        }))
+    ))
+}
+
+
+private func sideBarProps() -> [SideBarView.ButtonProps]{
```
Comment (AndriiMoskvin) [inline Demo/Demo/ContentControls.swift]: and here.

File: SnapshotTests/SnapshotTests.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -446,7 +446,7 @@
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 95F456P57Y;
+				DEVELOPMENT_TEAM = KYF8QFK3G9;
```
Comment (VladyslavAnokhin) [inline SnapshotTests/SnapshotTests.xcodeproj/project.pbxproj]: Are you sure about this changes?

Comment (RomanTysiachnik) [reply SnapshotTests/SnapshotTests.xcodeproj/project.pbxproj]: 100%

**Pull request merged**